src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -76,14 +80,14 @@ public Gated(Deadline deadline)
         /// </summary>
         public class Quarantined : EndpointPolicy
         {
-            public Quarantined(long uid, Deadline deadline)
+            public Quarantined(int uid, Deadline deadline)
```

Comment:
Changed this data type to `int`, which is what everything else uses. I don't know why it was inconsistent here.